nautobot_secrets_providers/providers/delinea.py

```diff
@@ -30,34 +30,34 @@
             SecretServerError,
         )
 
-        thycotic_installed = True  # pylint: disable=invalid-name
+        delinea_installed = True  # pylint: disable=invalid-name
     except ImportError:
-        thycotic_installed = False  # pylint: disable=invalid-name
+        delinea_installed = False  # pylint: disable=invalid-name
 
 from nautobot.core.forms import BootstrapMixin
 from nautobot.extras.secrets import exceptions, SecretsProvider
 
-from .choices import ThycoticSecretChoices
+from .choices import DelineaSecretChoices
 
 __all__ = (
-    "ThycoticSecretServerSecretsProviderId",
-    "ThycoticSecretServerSecretsProviderPath",
+    "DelineaSecretServerSecretsProviderId",
+    "DelineaSecretServerSecretsProviderPath",
 )
 
 
-class ThycoticSecretServerSecretsProviderBase(SecretsProvider):
-    """A secrets provider for Thycotic Secret Server."""
+class DelineaSecretServerSecretsProviderBase(SecretsProvider):
+    """A secrets provider for Delinea Secret Server."""
 
-    is_available = thycotic_installed
+    is_available = delinea_installed
 
     @classmethod
     def get_value_for_secret(cls, secret, obj=None, **kwargs):  # pylint: disable=too-many-locals
         """Return the value stored under the secret's key in the secret's path."""
-        # This is only required for Thycotic Secret Server therefore not defined in
+        # This is only required for Delinea Secret Server therefore not defined in
         # `required_settings` for the app config.
         plugin_settings = settings.PLUGINS_CONFIG["nautobot_secrets_providers"]
-        if "thycotic" not in plugin_settings:
-            raise exceptions.SecretProviderError(secret, cls, "Thycotic Secret Server is not configured!")
+        if "delinea" not in plugin_settings:
+            raise exceptions.SecretProviderError(secret, cls, "Delinea Secret Server is not configured!")
```

Comment:
The change to required plugin_settings is a breaking change, so this would need to be a major version bump to the app.